vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -139,11 +139,13 @@ public String getHeight() {
     @Override
     public void setWidth(String value) {
         width = value;
+        setDimension(ElementConstants.STYLE_WIDTH, value);
```

Comment:
Now that the API also updates the Web Component dimensions, the internal `resizeListener` should no longer use it but instead just update the `width` and `height` fields directly. Otherwise, users with slow connection will experience weird jumpy behavior.

Reply:
Change `resizeListener` to set properties instead.